SpectatorView/Calibration/Calibration/Main.cpp

```diff
@@ -16,6 +21,25 @@ LRESULT CALLBACK WndProc(HWND, UINT, WPARAM, LPARAM);
 // Entry point
 int WINAPI wWinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance, _In_ LPWSTR lpCmdLine, _In_ int nCmdShow)
 {
+
+	auto const filePath = CameraConfigurationFile::getMyDocumentPath() + "\\camera.cfg"s;
+
+	OutputDebugStringA(filePath.c_str());
+
+	try
+	{
+		CameraConfigurationFile cfg(filePath);
+		cfg.readConfig();
+
+		FrameProviderStaticConfig::width = int(cfg.getWidth());
+		FrameProviderStaticConfig::height = int(cfg.getHeight());
+		FrameProviderStaticConfig::fps = float(cfg.getFrameRate());
+	}
+	catch(const std::exception& e)
+	{
+		OutputDebugStringA(e.what());
```

Comment:
Is there a fallback to the previously defined constants?

Reply:
fixed